fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentPomTemplate.java

```diff
@@ -30,35 +30,10 @@ public static void setProject(FluentProject project) {
 
     @Override
     protected void writeBuildBlock(XmlBlock projectBlock, Pom pom) {
-        projectBlock.block("build", buildBlock -> {
-            buildBlock.block("plugins", pluginsBlock -> {
-                if (FluentStatic.getFluentJavaSettings().isSdkIntegration()) {
-                    // revapi-maven-plugin
-                    pluginsBlock.block("plugin", pluginBlock -> {
-                        pluginBlock.tag("groupId", "org.revapi");
-                        pluginBlock.tag("artifactId", "revapi-maven-plugin");
-                        pluginBlock.tagWithInlineComment("version", project.getPackageVersions().getRevapiMavenPlugin(),
-                                "{x-version-update;org.revapi:revapi-maven-plugin;external_dependency}");
-                        pluginBlock.block("configuration", configurationBlock -> {
-                            configurationBlock.block("analysisConfiguration", analysisConfigurationBlock -> {
-                                analysisConfigurationBlock.block("revapi.ignore", ignoreBlock -> {
-                                    ignoreBlock.block("item", itemBlock -> {
-                                        itemBlock.tag("code", "java.method.addedToInterface");
-                                    });
-
-                                    ignoreBlock.block("item", itemBlock -> {
-                                        itemBlock.tag("regex", "true");
-                                        itemBlock.tag("code", ".*");
-                                        itemBlock.tag("package", "com\\.azure\\.resourcemanager(\\.[^.]+)+\\.fluent(\\.[^.]+)*");
-                                    });
-                                });
-                            });
-                        });
-                    });
-                } else {
-                    writeStandAlonePlugins(pluginsBlock);
-                }
-            });
-        });
+        projectBlock.block("build", buildBlock -> buildBlock.block("plugins", pluginsBlock -> {
```

Comment:
We should check with @weidongxu-microsoft why they needed revapi to ignore breaking changes in `fluent` packages. If this is required, we should add this to the revapi exclusions file that is centrally maintained and not be generated for each library.

Reply:
It will be added centrally using this configuration:

```json
{
  "regex": true,
  "code": "java\\.method\\.addedToInterface",
  "new": "method .* com\\.azure\\.resourcemanager\\..*",
  "justification": "resourcemanager interfaces are allowed to add methods."
}
```

Reply:
The classes under `fluent` is usually the Inner classes that we do not expect user to use directly, but user can use them if they choose.

It might not much for Lite packages, as they would be exposed in interface anyway.

You can remove this, and we can see how it works.